### DISCUSSION TITLE
Logo for Quarto related software!

### DISCUSSION THREAD

**Discussion by ramiromagno** [quarto-dev/quarto-cli#6232] [Q&A]
### Description

In the same way that the hex shape became associated with R package logos, we should have something equivalent for Quarto related software, e.g. extensions.

I am suggesting this because I am putting together an extension and it makes me sad that I don't have a nice logo to put in the README.  :D

**Comment by mcanouil** [comment]
As for the hex logo, there is nothing that forbids you to do that.
I am not sure Quarto should tell users anything on that matter.

**Reply by ramiromagno** [reply]
Apologies if I was not clear. I don't mean Quarto to be prescriptive about logos :) I am suggesting that as an open source community that we could discuss it to create a trend :)

**Reply by ramiromagno** [reply]
For example, I just created my first extension, still very rough around the edges, but decided to add the Quarto simple logo to signal that it is related to Quarto: https://github.com/ramiromagno/wiley-njd.

**Reply by mcanouil** [reply]
I personally don't use logo for any of my 10+ extensions, but I changed the repository image for when I share my repositories.

![](https://github.com/mcanouil/mickael.canouil.fr/blob/main/projects/quarto-extension.png?raw=true)

**Reply by ramiromagno** [reply]
Thanks for sharing!
